proto/gds/models/v1/models.proto

```diff
@@ -3,8 +3,34 @@ syntax="proto3";
 package gds.models.v1;
 option go_package = "github.com/trisacrypto/directory/pkg/gds/models/v1;models";
 
+import "trisa/gds/models/v1beta1/ca.proto";
 import "trisa/gds/models/v1beta1/models.proto";
 
+// Certificate embeds a TRISA Certificate into a record that can be stored in the
+// database for certificate management.
+message Certificate {
+    // A unique identifier generated by the directory service for storage
+    string id = 1;
+
+    // CertificateRequest that this certificate is associated with
+    string request = 2;
+
+    // VASP that this certificate belongs to
+    string vasp = 3;
+
+    // Current status of the certificate
+    string status = 4;
+
+    // Certificate details
+    trisa.gds.models.v1beta1.Certificate details = 5;
+}
+
+enum CertificateState {
+    ISSUED = 0;
+    EXPIRED = 1;
+    REVOKED = 2;
+}
```

Comment:
I go back and forth on whether the zero-valued enum (which will be the defacto-default when initializing the Certificate struct) should be a "good" value (e.g. ISSUED probably is the default) or if should be a true zero-value, e.g. UNKNOWN to force the user to specify the status rather than make mistakes with the zero value. Thoughts?

```suggestion
enum CertificateState {
    UNKNOWN = 0;
    ISSUED  = 1;
    EXPIRED = 2;
    REVOKED = 3;
}
```

Reply:
Right, my thought was that GDS only needs to create these certificate records after they have already been issued, and there's no alternative initial state. However I do see the other side; it's a bit less clear from the code what state the certificate is in when it's created. I'm leaning towards keeping `ISSUED` as the initial state for right now but perhaps we should define a best practice for our future projects.